app/models/queue_item.rb

```diff
@@ -0,0 +1,18 @@
+class QueueItem < ActiveRecord::Base
+
+  validates_presence_of :user_id, :video_id
```

Comment:
It's a little weird for me. Since attributes in table `queue_items` named  user_id and video_id, why not use them?